tox.ini

```diff
@@ -110,6 +110,8 @@ usedevelop = True
 deps =
     -r requirements.txt
     -r dev_requirements/requirements-json_schemas.txt
+setenv =
+    TARGET_VERSION = {env:TARGET_VERSION:}
```

Comment:
geht das auf eine Variable in der `.env` zurück?

Reply:
Ne, die Variable wird im Workflow file gesetzt. Ich mach nen Kommentar dahin.